prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -109,7 +109,7 @@ if [[ $SYNCMODE == 1 ]]; then
   if has_detector_reco ITS; then
     [[ $RUNTYPE == "COSMICS" ]] && MFT_CONFIG_KEY+="MFTTracking.irFramesOnly=1;"
   fi
-  MFT_CONFIG_KEY+="MFTTracking.cutMultClusLow=0;MFTTracking.cutMultClusHigh=2000;"
+  MFT_CONFIG_KEY+="MFTTracking.cutMultClusLow=0;MFTTracking.cutMultClusHigh=4000;"
```

Comment:
@davidrohr I think this clashes with https://github.com/AliceO2Group/AliceO2/pull/11855